skipruntime-ts/Makefile

```
@@ -2,7 +2,9 @@ SHELL := /bin/bash
 
 SKARGO_PROFILE?=release
 NPM_WORKSPACES=$(shell jq --raw-output "[.workspaces[] | select((startswith(\"sql\") or contains(\"examples\")) | not)] | map(\"-w \" + .) | .[]" ../package.json)
+SKIPRUNTIME?=$(realpath $(shell dirname $(CURDIR)))/build/skipruntime
```

Just curious, but why not:
```suggestion
SKIPRUNTIME?=$(realpath $(CURDIR)/..)/build/skipruntime
```